type-check/pick.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as Test from "../src/types/Test";
+import { pick, toAsync } from "../src";
+
+const { checks, check } = Test;
+
+const obj1 = {
+  a: 1,
+  b: "2",
+  c: true,
+};
+
+type OptionalObj = {
+  a: number;
+  b?: string;
+};
+
+const res1 = pick(["a", "b"], obj1);
+const res2 = pick(["a", "c"], obj1);
+const res3 = pick(toAsync(["a", "b"] as const), obj1);
+const res4 = pick(["a", "b"], { a: 1 } as OptionalObj);
+const res5 = pick(["b", "c"], {
+  a: 1,
+  b: Symbol("b"),
+  c: Symbol.iterator,
+});
```

Comment:
I think `res5` test is not necessary, what is the difference from `res1`?

Reply:
I want to test for a type of `symbol`, but they seem to be the same case as you said. I merged both.